core/src/main/kotlin/org/evomaster/core/search/Individual.kt

```diff
@@ -162,9 +175,11 @@ abstract class Individual(override var trackOperator: TrackOperator? = null,
 
 
     open fun doInitialize(randomness: Randomness? = null){
-
-        seeAllActions()
-            .forEach { it.doInitialize(randomness) }
+        if (!areAllLocalIdsAssigned())
+            doInitializeLocalId()
+        if (!areAllGeneInitialized())
```

Comment:
not sure about this one... the point was to fail if we try to initialize something that is already initialized. when building an individual, shouldn't have actions/genes that are already initialized

Reply:
For `gene` and `action`, they can be randomized without an individual, 
but their local id can only be initialized once they are added to the individual.